modules/installation-minimum-resource-requirements.adoc

```diff
@@ -213,7 +213,7 @@ endif::ibm-power[]
 ifdef::azure[]
 [IMPORTANT]
 ====
-You are required to use Azure virtual machines with `premiumIO` set to `true`. The machines must also have the `hyperVGeneration` property contain `V1`.
+You are required to use Azure virtual machines with `premiumIO` set to `true`. The machines must also have the `hyperVGeneration` property contain `V2`.
```

Comment:
SME query: Do we have a resource that a user could visit in order to determine the correct version? Or maybe some sort of command or something or this nature?

Reply:
No requirement for the machines 'HyperVGeneration' property,  Possible values include: 'V1', 'V2'